R/edit.R

```diff
@@ -63,13 +67,27 @@ edit_git_ignore <- function(scope = c("user", "project")) {
 #' @param type Snippet type. One of "R", "markdown", "C_Cpp", "Tex",
 #'   "Javascript", "HTML", "SQL"
 edit_rstudio_snippets <- function(type = "R") {
+  create_directory("~", ".R")
```

Comment:
Why not move the directory creation logic into edit_file? (Also is there a recursive argument to create_directory?)